azure-batch/src/main/java/com/microsoft/azure/batch/BatchErrorCodeStrings.java

```diff
@@ -0,0 +1,353 @@
+/**
```

Comment:
@martinsawicki @anuchandy This is a fairly good set of error code definitions. We can figure out which ones are ARM common and start from there.